File: JAVA-Basic/JavaBasic/src/ConditionalStatements/ConditionalStatementsMoreExercises/Harvest.java
package ConditionalStatements.ConditionalStatementsMoreExercises;

import java.util.Scanner;

public class Harvest {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int x = Integer.parseInt(scan.nextLine());
        double y = Double.parseDouble(scan.nextLine());
        int z = Integer.parseInt(scan.nextLine());
        int workers = Integer.parseInt(scan.nextLine());

        double grapeForWine = (x * y) * 0.40;
        double wineProduction = grapeForWine / 2.5;

        if (wineProduction < z) {
            System.out.printf("It will be a tough winter! More %.0f liters wine needed.", Math.floor(z - wineProduction));
        } else {
            System.out.printf("Good harvest this year! Total wine: %.0f liters.\n", Math.floor(wineProduction));
            System.out.printf("%.0f liters left -> %.0f liters per person.", Math.ceil(wineProduction - z), Math.ceil((wineProduction - z) / workers));
        }
    }
}

File: 01.JS-Basic/More Exercises/PB - More Exercises/Conditional Statements/Harvest.js
function harvest(input) {

    let X = Number(input[0])  // •	1ви ред: X кв.м е лозето 
    let Y = Number(input[1])  // •	2ри ред: Y грозде за един кв.м 
    let Z = Number(input[2])  // •	3ти ред: Z нужни литри вино 
    let workers = Number(input[3]) // •	4ти ред: брой работници 

    let grapeProduction = X * Y

    let wineLitre = (grapeProduction * 0.40) / 2.5

    if (wineLitre < Z) {
        console.log(`It will be a tough winter! More ${Math.floor(Z - wineLitre)} liters wine needed.`)
    }
    else {
        console.log(`Good harvest this year! Total wine: ${Math.floor(wineLitre)} liters.`)
        console.log(`${Math.ceil(wineLitre - Z)} liters left -> ${Math.ceil((wineLitre - Z) / workers)} liters per person.`)
    }

}
harvest(["1020", "1.5", "425", "4"])